src/components/ThreeDNodeTree/stories/ThreeDNodeTree.stories.tsx

```diff
@@ -45,6 +52,82 @@ const setupMocks = () => {
   };
 };
 
+interface RightClickState {
+  visible: boolean;
+  rightClickedNode?: string;
+  menuStyle: {
+    [_: string]: string;
+  };
+}
+
+class RightClickExample extends React.Component<{}, RightClickState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      visible: false,
+      menuStyle: {},
+    };
+  }
+  showSubMenu = () => {
+    return this.state.visible ? (
+      <Menu
+        theme="dark"
+        style={this.state.menuStyle}
+        onClick={(params: ClickParam) => {
+          if (this.state.rightClickedNode) {
+            switch (params.key) {
+              case '1': {
+                alert('1: ' + this.state.rightClickedNode);
+                break;
+              }
+              case '2': {
+                alert('2: ' + this.state.rightClickedNode);
+                break;
+              }
+              default:
```

Comment:
Not necessary without any actions?

Reply:
Changed all actions into simply alerting names of the clicked nodes. 
Also enabled the component to distinguish clicks inside and outside the menu so that the context menu can be correctly closed when its outside is clicked and perform actions when its inner items are clicked.

Reply:
Yes but `default: break;` doesn't do anything. So can remove those two lines?

Reply:
It's already removed